test/helpers/getPerformanceTests.ts

```diff
@@ -0,0 +1,96 @@
+import { evaluateXPathToNodes, evaluateXPathToString, evaluateXPathToFirstNode } from 'fontoxpath';
+
+import testFs from './testFs';
+import {
+	ALL_TESTS_QUERY,
+	getAllTestSets,
+	getFile,
+	unrunnableTestCasesByName,
+} from './qt3TestsTools';
+
+const PERFORMANCE_TESTS_FILE = 'runnablePerformanceTestNames.csv';
+const TOTAL_NUMBER = 6420;
```

Comment:
Is this supposed to be the total number of QT3 tests? If so than wouldn't this mean that we have to update this each time we enable more tests?

Reply:
I changed the logic, now we want need that number